Source/Core/Core/HW/Wiimote.h

```diff
@@ -37,6 +37,7 @@ namespace Wiimote
 
 void Shutdown();
 void Initialize(void* const hwnd, bool wait = false);
+void ResetWiimotesForMovie();
```

Comment:
Would you please rename this function to ResetAllWiimotes() ?  It could be used for other purposes.